registration/register.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+
+	<head>
+
+	  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
+
+	  <title>Register For Spectra</title>
+
+	  <link rel="stylesheet" type="text/css" href="frontend/main.css"></head>
+
+	<body>
+
+	  <iframe id="typeform-full" width="100%" height="100%" frameborder="0" src="https://ingochris.typeform.com/to/gltwlZ"></iframe>
+
+	  <script type="text/javascript" src="https://s3-eu-west-1.amazonaws.com/share.typeform.com/embed.js"></script>
+
+	</body>
+
+</html>
```

Comment:
Try and keep indentation and spaces consistent with the other pages: https://github.com/SpectraHackathon/spectra-revamp/blob/master/index.html